packages/conf-liburing/conf-liburing.0.1.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Virtual package for system installation of liburing"
+maintainer: ["Lee Koon Wen"]
+authors: ["Lee Koon Wen"]
+license: ["ISC" "BSD-3-Clause"]
+homepage: "https://github.com/koonwen/uring-trace"
```

Comment:
To fix the missing lint

```
warning 37: Missing field 'dev-repo'
```

I think you may just add

```suggestion
dev-repo: "https://github.com/koonwen/uring-trace"
```

here and in the other package.